<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CovarianceDialog</class>
 <widget class="QDialog" name="CovarianceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>594</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Covariance matrix inspection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The &lt;span style='font-family: monospace'&gt;weight_matrix_focal&lt;/span&gt; used to solve the optimisation problem. A full, more accurate matrix can be calculated through &lt;span style='font-family: monospace'&gt;Covariance.weight_matrix&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutPlot"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignLeft">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Matrix selection</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="showWeightFocal">
          <property name="text">
           <string>weight_matrix_focal</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">matrixButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="showWeightFull">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Only available when full covariance matrix is calculated. See Tools &gt; Calculate full covariance/weight matrix.</string>
          </property>
          <property name="text">
           <string>weight_matrix</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">matrixButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="showCovarianceFocal">
          <property name="text">
           <string>covariance_matrix_focal</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">matrixButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="showCovarianceFull">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Only available when full covariance matrix is calculated. See Tools &gt; Calculate full covariance/weight matrix.</string>
          </property>
          <property name="text">
           <string>covariance_matrix</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">matrixButtonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>released()</signal>
   <receiver>CovarianceDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="matrixButtonGroup"/>
 </buttongroups>
</ui>
